packages/safe-ds-lang/src/language/generation/safe-ds-python-generator.ts

```diff
@@ -1020,7 +1020,12 @@ export class SafeDsPythonGenerator {
         );
         const fullyQualifiedTargetName = this.generateFullyQualifiedFunctionName(expression);
         if (!containsOptionalArgs && isSdsMemberAccess(expression.receiver)) {
-            const referenceImport = this.createImportDataForReference(expression.receiver.member!);
+            const classDeclaration = this.getOutermostContainerOfType(callable, isSdsClass)!;
+            const referenceImport = this.createImportDataForNode(
+                classDeclaration,
+                expression.receiver.member!,
+                classDeclaration.name,
```

Comment:
I'm still a bit doubtful about the `refName` (3rd parameter) here, but considering that the user has no way of specifying it here, that should be fine.

Reply:
Maybe we can make the 3rd parameter optional, with `declaration.name` as the default?